apps/src/templates/projects/SectionProjectsList.jsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import ProjectsList from './ProjectsList';
+import _ from 'lodash';
+import color from "../../util/color";
+
+const ALL_STUDENTS = '_all_students';
+
+const styles = {
+  filterDropdown: {
+    margin: 0,
+    color: 'dimgray'
+  },
+  filterRow: {
+    backgroundColor: color.teal,
+    borderBottom: 'solid 1px white',
+    color: color.white,
+    padding: 10,
+    fontSize: 14
+  },
+  filterSpan: {
+    float: 'right',
+    display: 'inline-flex',
+    alignItems: 'center',
+  },
+  clearDiv: {
+    clear: 'both'
+  }
+};
+
+class SectionProjectsList extends React.Component {
+  constructor(props) {
+    super(props);
+
+    const studentNames = this.getStudentNames(props.projectsData);
+
+    this.state = {
+      studentNames,
+      selectedStudent: ALL_STUDENTS,
+    };
+  }
+
+  getStudentNames(projectsData) {
+    const studentsMap = {};
+    projectsData.forEach(project => {
+      studentsMap[project['studentName']] = true;
+    });
+    return Object.keys(studentsMap);
+  }
+
+  onChangeStudent(event) {
+    const selectedStudent = event.target.value;
+    this.setState({selectedStudent});
+  }
+
+  componentWillReceiveProps(nextProps) {
+    if (_.isEqual(nextProps, this.props)) {
+      return;
+    }
+
+    const studentNames = this.getStudentNames(nextProps.projectsData);
+    this.setState({studentNames});
+  }
+
+  render() {
+    const filteredProjectsData = this.props.projectsData.filter(project => (
+      [ALL_STUDENTS, project['studentName']].includes(this.state.selectedStudent)
+    ));
+
+    return (
+      <div>
+        <div style={styles.filterRow}>
+          <span style={styles.filterSpan}>
```

Comment:
Optional: It _might_ be useful to extract a `StudentFilterDropdown` component, probably this `span` down, that follows the stateless input pattern and could take a little of the nitty-gritty display logic out of this wrapper component - it might be helpful down the road if we add additional filters to this filter row.  Or that could wait until there's more than one filter.  Your call.